CorsixTH/Lua/filesystem.lua

```diff
@@ -76,12 +76,13 @@ end
 
 --! Set the root physical path for this FileSystem.
 -- If the path is an ISO then set the provider. If the path is a directory
--- then set the pysical_path and populate the files and sub_dirs.
+-- then set the physical_path and populate the files and sub_dirs.
 --
 --!param physical_path (string) a path on the filesystem to either a directory
 -- or theme hospital ISO file.
 function FileSystem:setRoot(physical_path)
-  if physical_path:match"%.[iI][sS][oO]$" or physical_path:match"%.[iI][sS][oO]9660$" then
+  local path = physical_path:lower()
+  if path:match("%.iso$") or path:match("%.iso9660$") or path:match("%.dmg$") then
```

Comment:
Any easy way to share this check with the filesystem check for the same thing? (I like the way it was done there a little better, if you had to choose implementation)